build/illume.js

```diff
@@ -11,17 +11,15 @@ var _q = require("@standard-library/q");
 var _kefir = require("kefir");
 
 var offsetAbove = function offsetAbove(y, element) {
-  return y > $(element).offset().top;
+  return y > element.offsetTop;
```

Comment:
I pushed some things from yesterday (that I halfway-remember doing, heh), and I set up tests to run for different browsers:

https://github.com/standard-library/illume/blob/99f5e4ad6cf292eaa254e3b4ead9442d39527756/package.json#L8

Not sure if that's the right approach?

JS has not been corrected yet because I wanted (at the time) to get the tests working.
https://github.com/standard-library/illume/blob/99f5e4ad6cf292eaa254e3b4ead9442d39527756/test/illume.js

This is what `illume("name")` looks like when I `console.log` it in the browser. Is this what it should be looking like?
![Uploading Screen Shot 2017-05-02 at 9.55.40 AM.png…]()